### PR TITLE
build : versioning and tainted build version

### PR DESCRIPTION
**Pull request by nplanel**

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -49,12 +51,12 @@ all: install
 	gofmt -w -s statics/bindata.go
 
 .compile:
-	${GOPATH}/bin/govendor install ${GOFLAGS} ${GOTAGS} ${VERBOSE_FLAGS} +local
+	${GOPATH}/bin/govendor install -ldflags="-X main.Version=${VERSION}" ${GOFLAGS} ${GOTAGS} ${VERBOSE_FLAGS} +local
```
Comment (lebauce) [inline Makefile]: ${GOPATH}/bin/govendor install -ldflags="-X github.com/skydive-project/skydive/version.Version=${VERSION}" ${GOFLAGS} ${GOTAGS} ${VERBOSE_FLAGS} +local

File: Makefile [hunk 2]
```diff
@@ -49,12 +51,12 @@ all: install
...
 
 install: govendor genlocalfiles dpdk.build contribs .compile
 
 build: govendor genlocalfiles dpdk.build contribs
-	${GOPATH}/bin/govendor build ${GOFLAGS} ${GOTAGS} ${VERBOSE_FLAGS} +local
+	${GOPATH}/bin/govendor build -ldflags="-X main.Version=${VERSION}" ${GOFLAGS} ${GOTAGS} ${VERBOSE_FLAGS} +local
```
Comment (lebauce) [inline Makefile]: ${GOPATH}/bin/govendor build -ldflags="-X github.com/skydive-project/skydive/version.Version=${VERSION}" ${GOFLAGS} ${GOTAGS} ${VERBOSE_FLAGS} +local

File: skydive.go [hunk 3]
```diff
@@ -22,8 +22,14 @@
 
 package main
 
-import "github.com/skydive-project/skydive/cmd"
+import (
+	"github.com/skydive-project/skydive/cmd"
+	"github.com/skydive-project/skydive/version"
+)
+
+var Version = "currentVersion"
```
Comment (lebauce) [inline skydive.go]: Can be removed

File: skydive.go [hunk 4]
```diff
@@ -22,8 +22,14 @@
...
 
 func main() {
+	version.Version = Version
```
Comment (lebauce) [inline skydive.go]: Can be removed

File: Makefile (hunk 1 above)
Comment (nplanel) [reply Makefile]: indeed

**Pull request merged**